src/otx/recipe/anomaly_classification/uflow.yaml

```diff
@@ -0,0 +1,44 @@
+model:
```

Comment:
I think we only need `recipe/anomaly`. anomaly_classification/seg/det are going to be removed